packages/components/src/templates/next/components/complex/CollectionWidget/CollectionWidget.tsx

```diff
@@ -0,0 +1,199 @@
+    cardLastUpdated: "prose-label-sm-medium text-base-content",
+    cardTitle:
+      "prose-headline-lg-semibold text-base-content-strong group-hover:text-brand-canvas-inverse",
+    cardTitleArrow:
+      "mb-0.5 ml-1 inline h-auto w-6 transition ease-in group-hover:translate-x-1",
+    cardCategory: "prose-label-sm-regular text-base-content-light",
+    urlButtonContainer: "mx-auto block",
+  },
+  variants: {
+    isExternalLink: {
+      true: {
+        cardTitleArrow: "rotate-[-45deg]",
+      },
+    },
+    numberOfCards: {
+      1: {
+        grid: "",
+      },
+      2: {
+        grid: "md:grid-cols-2",
+      },
+      3: {
+        grid: "md:grid-cols-2 lg:grid-cols-3",
+      },
+    },
+  },
+  defaultVariants: {
+    numberOfCards: NUMBER_OF_PAGES_TO_DISPLAY,
+  },
+})
+
+const compoundStyles = createInfoCardsStyles()
+
+const SingleCard = ({
+  title,
+  image,
+  category,
+  referenceLinkHref,
+  lastUpdated,
+  displayThumbnail,
+  displayCategory,
+  site,
+  LinkComponent,
+  shouldLazyLoad,
+}: CollectionWidgetSingleCardProps): JSX.Element => {
+  const isExternalLink = !!referenceLinkHref && isExternalUrl(referenceLinkHref)
+
+  return (
+    <Link
+      href={referenceLinkHref}
+      className={compoundStyles.cardContainer()}
+      LinkComponent={LinkComponent}
+      isExternal={isExternalLink}
+    >
+      {/* TODO: Add fallback image if displayThumbnail but image is not present e.g. link variant */}
+      {displayThumbnail && image && (
+        <div className={compoundStyles.cardImageContainer()}>
+          <ImageClient
+            src={
+              isExternalUrl(image.src) || site.assetsBaseUrl === undefined
+                ? image.src
+                : `${site.assetsBaseUrl}${image.src}`
+            }
+            alt={image.alt}
+            width="100%"
+            className={compoundStyles.cardImage()}
+            lazyLoading={shouldLazyLoad}
+            assetsBaseUrl={site.assetsBaseUrl}
+          />
+        </div>
+      )}
+
+      <div className={compoundStyles.cardTextContainer()}>
+        {lastUpdated && (
+          <p className={compoundStyles.cardLastUpdated()}>
+            {getFormattedDate(lastUpdated)}
+          </p>
+        )}
+
+        <h3 className={compoundStyles.cardTitle()}>
+          {title}
+          <BiRightArrowAlt
+            aria-hidden
+            className={compoundStyles.cardTitleArrow({
+              isExternalLink,
+            })}
+          />
+        </h3>
+
+        {displayCategory && category && (
+          <p className={compoundStyles.cardCategory()}>{category}</p>
+        )}
+      </div>
+    </Link>
+  )
+}
+
+export const CollectionWidget = ({
+  site,
+  LinkComponent,
+  collectionReferenceLink,
+  customTitle,
+  customDescription,
+  displayThumbnail,
+  displayCategory,
+  buttonLabel,
+  shouldLazyLoad,
+}: CollectionWidgetProps): JSX.Element => {
+  const collectionId = getResourceIdFromReferenceLink(collectionReferenceLink)
```

Comment:
Thought about adding a check that the siteId and collectionId indeed belongs to this site, but decided to skip it as am thinking this error will only happen by manual error by isomer team edits